src/encoded/static/components/static-pages/placeholders/BasicCarousel.js

```diff
@@ -76,11 +83,27 @@ export class BasicCarousel extends React.PureComponent {
         }, 10);
     }
 
+    static allSlidesHaveDimensions(slides = []){
+        const slidesLen = slides.length;
+        for (var i = 0; i < slidesLen; i++){
+            // When const/let are used inside a for loop in ES6+,
+            // new function/closure is created.
```

Comment:
Random info / probably shouldn't be in comment.
If checkout how gets compiled, the contents of for loop when use const or let within it (unless var is used/reused as in 'raw javascript'), becomes a function in the resulting 'compiled raw javascript' to avoids re-using same const variable for example in subsequent iteration of the for loop.